airflow/www/views.py

```diff
@@ -3559,11 +3572,10 @@ def datasets_summary(self):
                     if session.bind.dialect.name == "postgresql":
                         order_by = (order_by[0].nulls_first(), *order_by[1:])
 
-            total_entries = session.query(func.count(DatasetModel.id)).scalar()
+            count_query = session.query(func.count(DatasetModel.id))
```

Comment:
since you are not doing a group by in the count query, you need to do count distinct

Reply:
@bbovenzi For an example, see how we count the number of dataset events in the results query above.